flowforge/parsers/OutputParser.py

```diff
@@ -20,4 +20,4 @@ def parse(self) -> List[Any]:
         List[Any]
             The parsed output
         """
-        pass
\ No newline at end of file
+        return
```

Comment:
what's the logic to this change?

Reply:
Pylint gave an error for using `pass` here.